src/backends/plotly.jl

```diff
@@ -1019,14 +1019,16 @@ function plotly_polar!(plotattributes_out::KW, series::Series)
     end
 end
 
-function plotly_hover!(plotattributes_out::KW, hover)
-    # hover text
-    if hover === nothing || all(in([:none, false]), hover)
+function plotly_adjust_hover_label!(plotattributes_out::KW, hover)
+    if isnothing(hover)
```

Comment:
```suggestion
    if hover === nothing
```

Reply:
Conventions. We're fine with using either one of the two but not both. Code style must be consistent.